### PR TITLE
Feat(signup.hbs): Loader added post form submittion

### PR DESCRIPTION
**Pull request by SagarBajpai**

### CODE REVIEW

File: app/controllers/signup.js [hunk 1]
```diff
@@ -223,10 +225,12 @@ export default class SignupController extends Controller {
       if (status === 204) {
         window.open('https://realdevsquad.com/goto', '_self');
       } else {
-        alert('Something went wrong. Please check console errors.');
+        // alert('Something went wrong. Please check console errors.');
       }
```
Comment (swarajpure) [inline app/controllers/signup.js:229]: Do we need this else statement if there's nothing anyway in there?

**Review by swarajpure — CHANGES_REQUESTED** [review]
Such a simple approach 😄
I like that.

Suggesting some change, please check

File: app/controllers/signup.js (hunk 1 above)
Comment (SagarBajpai) [reply app/controllers/signup.js:229]: Ohhh, my bad, the code needs to be uncommented.

File: app/index.html [hunk 2]
```diff
@@ -11,7 +11,7 @@
 
     <link integrity="" rel="stylesheet" href="{{rootURL}}assets/vendor.css">
     <link integrity="" rel="stylesheet" href="{{rootURL}}assets/website-my.css">
-
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
[the remark this answers is not in the document]
Comment (SagarBajpai) [reply app/index.html:14]: I'm suggesting here, Let's use the CDN for now:
As we are not using a lot of icons as of now, using the library will include the library in the final build. On the other side CDN will be simply cached.
Later on, we can use the library.

Also when you'll merge your PR, rebase will solve the conflict.
Let me know your views

[the remark this answers is not in the document]
Comment (swarajpure) [reply app/index.html:14]: I'm not really sure what to choose here 😅 
@ankushdharkar @sumitd94 What would you suggest?

[the remark this answers is not in the document]
Comment (swarajpure) [reply app/index.html:14]: As we discussed, we'll go with the library (reference: tree-shaking). 
Please check the link in the above comments, we'll be using that library for the same.

**Review by swarajpure — CHANGES_REQUESTED** [review]
Please solve the conflicts

**Review by swarajpure — APPROVED** [review]
Once PR #31 is merged, font-awesome icons will be coming from the library

Merging this for now.

Thank you for implementing the loader in such a simple, elegant way 😉

**Pull request merged**